src/org/usfirst/frc2175/subsystem/intake/DreamIntakeSubsystem.java

```diff
@@ -4,12 +4,15 @@
 import org.usfirst.frc2175.config.WiringConfig;
 import org.usfirst.frc2175.subsystem.BaseSubsystem;
 
+import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.Talon;
 
 public class DreamIntakeSubsystem extends BaseSubsystem {
     private Talon dreamIntakeSideBeltTalon;
     private Talon dreamIntakeMainBeltTalon;
     private Talon dreamIntakeLiftTalon;
+    private DigitalInput dreamIntakeUpSwitch;
+    private DigitalInput dreamIntakeDownSwitch;
```

Comment:
You forgot to set the value of these variables again. You need to add stuff like this to the constructor: `this.dreamIntakeUpSwitch = wiringConfig.getDreamIntakeUpSwitch();`.